backend/test/v2/api/integration_suite_test.go

```diff
@@ -0,0 +1,132 @@
+package api
+
+import (
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"k8s.io/client-go/kubernetes"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	api_server "github.com/kubeflow/pipelines/backend/src/common/client/api_server/v2"
+	"github.com/kubeflow/pipelines/backend/test/v2/api/logger"
+	test "github.com/kubeflow/pipelines/backend/test/v2/api/utils"
+	utils "github.com/kubeflow/pipelines/backend/test/v2/api/utils"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
+	. "github.com/onsi/gomega"
+)
+
+var k8Client *kubernetes.Clientset
+var pipelineUploadClient *api_server.PipelineUploadClient
+var pipelineClient *api_server.PipelineClient
+var testStartTimeUTC time.Time
+var testLogsDirectory = "logs"
+var testReportDirectory = "reports"
+var junitReportFilename = "junit.xml"
+var jsonReportFilename = "report.json"
+
+var _ = BeforeSuite(func() {
+	err := os.MkdirAll(testLogsDirectory, 0755)
+	if err != nil {
```

Comment:
Just curious- any reason for logging the errors and returning here instead of using Expect(err).NotTo(HaveOccurred())?